### PR TITLE
Feature/1 reusable button

### PR DESCRIPTION
**Pull request by Jyshe**
Added className props and Icon component props to the Header Component.

### CODE REVIEW

File: src/Components/Header/index.tsx [hunk 1]
```diff
@@ -1,16 +1,25 @@
 
-import { LeftArrow } from '../../assets/Icons'
-
-export type HeroProps = {
+export type HeaderProps = {
```
Comment (chandran-kannan) [inline src/Components/Header/index.tsx]: why are you exporting types?

File: src/Components/Header/index.tsx [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
+      <span>
+        {leftIcon}
+      </span>
+     <h6 className={`font-600 fs-18px text-black header-shadow ${(className)}`} > {title} </h6>
+     <span>
+      {rightIcon}
+     </span> 
+   </div>
+
+   /*To use it => <Header title='Sign Up' leftIcon={<LeftArrow/>} className="text-red"/>*/
```
Comment (chandran-kannan) [inline src/Components/Header/index.tsx]: move the comment to below the type declaration

File: src/assets/Icons/index.tsx [hunk 3]
```diff
@@ -1,9 +1,18 @@
 
-export const LeftArrow = () => {
-  return (
-    <svg width="24" height="18" viewBox="0 0 24 18" fill="black" xmlns="http://www.w3.org/2000/svg">
-<path d="M23 7.99999H2.58998L7.99998 2.63999C8.09371 2.54703 8.16811 2.43643 8.21888 2.31457C8.26964 2.19271 8.29578 2.062 8.29578 1.92999C8.29578 1.79798 8.26964 1.66727 8.21888 1.54542C8.16811 1.42356 8.09371 1.31296 7.99998 1.21999C7.81262 1.03374 7.55917 0.929199 7.29498 0.929199C7.0308 0.929199 6.77735 1.03374 6.58998 1.21999L0.879985 6.87999C0.318183 7.44249 0.00262451 8.20499 0.00262451 8.99999C0.00262451 9.795 0.318183 10.5575 0.879985 11.12L6.53998 16.78C6.72625 16.9647 6.97764 17.0689 7.23998 17.07C7.37159 17.0708 7.50205 17.0455 7.62389 16.9958C7.74573 16.946 7.85654 16.8727 7.94998 16.78C8.04371 16.687 8.11811 16.5764 8.16888 16.4546C8.21964 16.3327 8.24578 16.202 8.24578 16.07C8.24578 15.938 8.21964 15.8073 8.16888 15.6854C8.11811 15.5636 8.04371 15.453 7.94998 15.36L2.58998 9.99999H23C23.2652 9.99999 23.5196 9.89464 23.7071 9.7071C23.8946 9.51956 24 9.26521 24 8.99999C24 8.73478 23.8946 8.48042 23.7071 8.29289C23.5196 8.10535 23.2652 7.99999 23 7.99999Z" fill="black"/>
-</svg>
-  )
-}
+type IconProps = {
```
Comment (chandran-kannan) [inline src/assets/Icons/index.tsx:2]: good

**Review by chandran-kannan — CHANGES_REQUESTED** [review]
update the changes

File: src/Components/Header/index.tsx (hunk 1 above)
Comment (Jyshe) [reply src/Components/Header/index.tsx]: removed the export this type will be local to the component

File: src/Components/Header/index.tsx (hunk 2 above)
Comment (Jyshe) [reply src/Components/Header/index.tsx]: did the changes

**Review by chandran-kannan — APPROVED** [review]

**Pull request merged**